tools/fix_auth/cli.rb

```diff
@@ -7,8 +7,8 @@ class Cli
     def parse(args, env = {})
       args.shift if args.first == "--" # Handle when called through script/runner
       self.options = Optimist.options(args) do
-        banner "Usage: ruby #{$PROGRAM_NAME} [options] database [...]\n" \
-               "       ruby #{$PROGRAM_NAME} [options] -P new_password database [...] to replace all passwords"
+        banner "Usage: #{$PROGRAM_NAME} [options] database\n" \
+               "       #{$PROGRAM_NAME} [options] -P new_password database to replace all passwords"
```

Comment:
Since this is showing usage, the "to replace all passwords" part gets mixed up with the requried parameters...how about

```suggestion
               "       #{$PROGRAM_NAME} [options] -P new_password database   # to replace all passwords"
```

Reply:
I added a few examples here. hope it is more clear.

Reply:
I don't see the changes? Forgot to push maybe?

That being said I don't think we need more examples - I only suggested separating the "to replace all passwords" description from the actual parameters with a separator.

Reply:
yup, forgot to push